projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-comments-tree-endpoint.php

```diff
@@ -140,22 +147,26 @@ function get_comment_db_status( $status ) {
 	/**
 	 * Determine if the passed comment status is valid or not.
 	 *
-	 * @param string $status
+	 * @param string $status - comment status.
 	 *
 	 * @return boolean
 	 */
-	function validate_status_param( $status ) {
+	public function validate_status_param( $status ) {
+		// @todo see if we can use a strict comparison here.
+		// phpcs:ignore WordPress.PHP.StrictInArray.MissingTrueStrict
 		return in_array( $status, array( 'all', 'approved', 'unapproved', 'pending', 'spam', 'trash' ) );
```

Comment:
We should be able to add a true here. We do want the `$status` to match one of those, so we do want to compare a string to a string.